app/assets/javascripts/pension-pot.js

```diff
@@ -36,8 +36,8 @@
       this.$numberInputs.on('keypress', function (e) {
         //if the letter is not digit then display error and don't type anything
         if (e.which !== 8 && e.which !== 0 && (e.which < 48 || e.which > 57)) {
-          //display error message
-          $('#errmsg').html('Only use numbers').show().fadeOut('slow');
+          var error_input = $(e.target).parents('.form-group').find('.js-error-message');
+          error_input.html('Only use numbers').show().fadeOut('slow');
```

Comment:
done and duplicated in [output](https://github.com/guidance-guarantee-programme/output/pull/215/commits/c4781f514f5c06a24efe815f787979ce8b1c31be)